core/templates/dev/head/css/oppia.css

```diff
@@ -3590,6 +3590,12 @@ md-card.preview-conversation-skin-supplemental-card {
     text-decoration: underline;
 }
 
+.oppia-dashboard-list-summary {
+  display: block;
+  color: inherit !important;
```

Comment:
I am not able to find another way. The above rules always supersede this rule resulting in green color. Could you suggest any way?